src/component/Roadmap_Jsondata.jsx

```diff
@@ -0,0 +1,4209 @@
+    "category": ["Software Operations"],
+    "subcategory": ["Forensics"],
+    "level": "Intermediate",
+    "provider": "CREST",
+  },
+  {
+    "name": "CDRP",
+    "url": "https://app.infosecinstitute.com/portal/courses/a0tC0000000FovhIAC",
+    "description": "Learn how to recover sensitive data from damaged or partially destroyed hard drives, solid-state media and removable media! A combination of hands-on labs and expert instruction builds your knowledge around data recovery and prepares you to become a Certified Data Recovery Professional (CDRP).Award-Winning Training.For 16 years Infosec has been one of the most awarded and trusted information security training vendors — 40+ industry awards!.Exam Pass Guarantee.We offer peace of mind with our Exam Pass Guarantee.Track Record of Success.Infosec training courses have received 12,000+ 5-star reviews from our students!",
+    "category": ["Software Operations"],
+    "subcategory": ["Forensics"],
+    "level": "Intermediate",
+    "provider": "Infosec Skills",
+  },
+  {
+    "name": "eCDFP",
+    "url": "https://security.ine.com/certifications/ecdfp-certification/",
+    "description": "eCDFP Certification.Certified Digital Forensics Professional.eCDFP is an advanced digital forensics exam meant for senior-level Cybersecurity professionals. A successful certification allows digital forensics investigators to prove their technical digital forensics expertise.",
+    "category": ["Software Operations"],
+    "subcategory": ["Forensics"],
+    "level": "Intermediate",
+    "provider": "iNE Security",
+  },
+  {
+    "name": "CMFE",
+    "url": "https://app.infosecinstitute.com/portal/courses/a0t1A000009H6juQAC",
+    "description": "Learn how to use mobile forensics to investigate cybercrime! Our Mobile Forensics Boot Camp builds your skills in a hands-on lab environment so you can apply what you learned the day you leave training.Award-Winning Training For 16 years Infosec has been one of the most awarded and trusted information security training vendors — 40+ industry awards!Exam Pass Guarantee.We offer peace of mind with our Exam Pass Guarantee.Track Record of Success.Infosec training courses have received 12,000+ 5-star reviews from our students!",
+    "category": ["Software Operations"],
+    "subcategory": [
```

Comment:
Change the provide to **`Dark Vortex`**

Reply:
No, this is the provider "Infosec Skills"

Reply:
Yes, that is correct, I don't know why the comment got repeated on line no **3107**, you can mark them resolved!
The comment was orignally only meant to be for https://github.com/Golden-Wave-Tech/SecCertRoadmap/pull/2#discussion_r1781550033